code/__HELPERS/game.dm

```diff
@@ -187,3 +187,25 @@
 	source.luminosity = lum
 
 	return heard
+
+/proc/get_active_player_count(alive_check = 0, afk_check = 0, human_check = 0)
+	// Get active players who are playing in the round
+	var/active_players = 0
+	for(var/i = 1; i <= GLOB.player_list.len; i++)
+		var/mob/M = GLOB.player_list[i]
+		if(!(M && M.client))
+			continue
+		if(alive_check && M.stat)
+			continue
+		else if(afk_check && M.client.is_afk())
+			continue
+		else if(human_check && !ishuman(M))
+			continue
```

Comment:
To make this a little more generic you could do this as a faction check instead.
Have the default be empty and only check if filled, but we could then do FACTION_MARINE vs FACTION_XENO or even `FACTION_NEUTRAL`